Framework/Backend/services/jira.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+*/
+
+const https = require('https');
+const log = new (require('./../log/Log.js'))('JIRA');
+
+/**
+ * Handles creating JIRA issues
+ */
+class Jira {
+  /**
+   * @param {object} config - JIRA configuration including URL, service account and project ID
+   */
+  constructor(config) {
+    if (!config) {
+      throw new Error('Configuration object cannot be empty');
+    }
+    if (!config.url) {
+      throw new Error('JIRA URL must be defined');
+    }
+    if (!config.serviceAccount || !config.serviceAccount.user || !config.serviceAccount.pass) {
+      throw new Error('Service account for JIRA must be defined');
+    }
+    if (config.projectId) {
```

Comment:
I believe here the condition should be negated. To prevent this in the future, it would be useful to have a test covering this case